crates/but-workspace/tests/fixtures/scenario/insert-blank-commit-three-commits copy.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -eu -o pipefail
+
+echo "/remote/" > .gitignore
+mkdir remote
+pushd remote
+git init
+popd
+
+git init
+
+git remote add origin ./remote
+
+git checkout -b one
+echo "foo" > one.txt
```

Comment:
How is files relevant as initial state to adding blank commits.
Maybe it's copied from a very old fixture from somewhere? I really thought we could do more with less by now, using function in `shared.sh`.